src/plugins/visualizations/public/visualize_app/components/viz_chart_warning.tsx

```diff
@@ -149,11 +149,21 @@ const TimelionWarningFormatMessage: FC<WarningMessageProps> = (props) => {
   );
 };
 
+const ControlsWarningFormatMessage: FC<WarningMessageProps> = (props) => {
+  return (
+    <FormattedMessage
+      id="visualizations.controls.notificationMessage"
+      defaultMessage="Input controls are deprecated and will be removed in a future release. Use the new Controls to filter and interact with your dashboard data. "
```

Comment:
Is there any documentation for the new controls to show to the users? This is your call of course but I think, it would be better to have a link to a documentation for the new controls. How do I enable them? Where can I find them etc

Reply:
@KOTungseth Do we have a link for the new controls docs yet? If not, I think we could safely add this after feature freeze as well as long as we can get the actual deprecation code in first :)